src/firefly/js/templates/lightcurve/ptf/PTFMissionOptions.js

```diff
@@ -149,17 +149,19 @@ export function ptfOnFieldUpdate(fieldKey, value) {
 }
 
 /**
- *
- * Gets the download option panel for PTF with specific file processor id 'PtfDownload'
+ *  This is specialized for PTF download.
+ *  Gets the download option panel for PTF with specific file processor id 'PtfDownload'
  * @param mission
  * @param cutoutSizeInDeg
  * @returns {XML}
  */
 export function ptfDownloaderOptPanel (mission, cutoutSizeInDeg) {
-
+    const currentTime = Date();
+    const style = {width: 247};
     return (
         <DownloadButton>
             <DownloadOptionPanel
+                groupKe = {mission}
```

Comment:
'groupKey' is misspelled. I'm not sure is used but you probably want to fix the name.

Reply:
It is a typo.  I will fix it.